cmd/collector/Dockerfile

```diff
@@ -1,5 +1,11 @@
+FROM alpine as certs
+RUN apk add --update --no-cache ca-certificates
+
 FROM scratch
 
+COPY --from=certs /usr/share/ca-certificates/ /usr/share/ca-certificates/
+COPY --from=certs /etc/ssl/ /etc/ssl/
```

Comment:
Do these directories need to be copied entirely? 
Can't we get away by copying just `/etc/ssl/certs/ca-certificates.crt` instead, which contains a single file version of the CA certificates?